openfisca_us/parameters/benefit/NSLP/benefit.yaml

```diff
@@ -0,0 +1,14 @@
+description: Annual value of NSLP per child
+
+2021-07-16: 595.8
```

Comment:
Nice - not a breaking issue but could we add parameters covering the start of 2021 and 2021? I had a look at the second reference (NSLP - Rates of Reimbursement) but I couldn't seem to find the values, or they weren't loading.